app/.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
```

Comment:
what is this star?

Reply:
Asterisk mark here means "all files," so instructions in this section will be applied for every files in the project. If you want to override some instructions for specific files, e.g. Javascript files, then you may append `[*.js]` section to the end of the file.

Reply:
so comfortable mark it is! thank you 😄